websites/serializers.py

```diff
@@ -320,9 +333,8 @@ class WebsiteStatusSerializer(
     WebsiteGoogleDriveMixin,
     WebsiteValidationMixin,
     WebsiteUrlSuggestionMixin,
+    WebsiteHasMetadataMixin,
 ):
-    """Serializer for website status fields"""
-
```

Comment:
Was there a reason for removing this docstring?

Reply:
No, removed by mistake